pkg/cmd/show.go

```diff
@@ -407,7 +407,7 @@ func showTerraform(name string) {
 	count := 0
 	for _, pod := range pods.Items {
 		if strings.Contains(pod.Name, name) && pod.Status.Phase == "Running" {
-			output, err = ExecCmdReturnOutput("bash", "-c", "export KUBECONFIG="+KUBECONFIG+"; kubectl get pods "+pod.Name+" -o wide -n "+pod.Namespace)
+			output, err = ExecCmdReturnOutput("kubectl", "--kubeconfig="+KUBECONFIG, "get", "pods", pod.Name, "-o wide", "-n "+pod.Namespace)
```

Comment:
```go
output, err = ExecCmdReturnOutput("kubectl", "--kubeconfig="+KUBECONFIG, "get", "pods", pod.Name, "-o wide", "-n "+pod.Namespace)
```
I guess this won't work, right?
either
- ` "-owide", "-n"+pod.Namespace` (no space)
- ` "-o", "wide", "-n", pod.Namespace` (or separate argument with comma)

Reply:
yes you are right, thanks! i fixed this and also include some missing parts